app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+
+const {
+    getCategories
+} = require('./controllers/categories');
+
+app.get('/api/categories', getCategories);
+
+app.all('/api/*',(req, res) => {
+    res.status(404).send({ msg: 'Route not found'})
+})
```

Comment:
I would probably change this to '/*' rather than '/api/*' as then it will cover any route not handled above rather than just those that begin with api!